### PR TITLE
[APIS-900] Add Autocommit mode to ODBC Data Source Setting window

### PR DESCRIPTION
**Pull request by airnet73**
http://jira.cubrid.org/browse/APIS-900

Purpose
ODBC driver always works with autocommit=false. (Cannot be changed)
In this state, if more than 100 queries are performed in one transaction, an error occurs.To solve this problem, Autocommit mode should be added to the ODBC setting window so that the user can set it himself.

Error message:
[-830]Cannot allocate query entry any more. Maximum allocatable entries are 100

Implementation
N/A

Remarks
N/A

### CODE REVIEW

File: odbc_connection.c [hunk 1]
```diff
@@ -817,6 +818,17 @@ odbc_connect_new (ODBC_CONNECTION * conn,
   pt = charset == NULL ? "" : charset;
   conn->charset = UT_MAKE_STRING (pt, -1);
 
+  ut_tolower (autocommit);
+  pt = autocommit == NULL ? "on" : autocommit;
+  if (strcmp (autocommit, "on") == 0)
```
Comment (kisoo-han) [inline odbc_connection.c]: Suggestion: remove function **ut_tolower ()** and use case independent string compare,  **_strcmp(),** as other codes do, if it is possible.

```
  pt = autocommit == NULL ? "on" : autocommit;
  if (_strcmp (autocommit, "on") == 0)
```

**Review by kisoo-han — APPROVED** [review]

**Comment by kisoo-han** [comment]
It would be nice to pass the test build to the ERWIN for feedback.

File: odbc_connection.c (hunk 1 above)
Comment (airnet73) [reply odbc_connection.c]: ut_tolower() is to compare the strings input in AUTOCOMMIT without case-sensitive.
Are you saying it should be case sensitive?

Comment (kisoo-han) [reply odbc_connection.c]: **_strcmp()** is case insensitive string compare.

Comment (kisoo-han) [reply odbc_connection.c]: My mistake it was
**_stricmp ()**

Comment (kisoo-han) [reply odbc_connection.c]:
```
  pt = autocommit == NULL ? "on" : autocommit;
  if (_stricmp (autocommit, "on") == 0)
```

**Pull request merged**